test/recipes/70-test_sslmessages.t

```diff
@@ -358,3 +369,16 @@ checkhandshake($proxy, checkhandshake::DEFAULT_HANDSHAKE,
                checkhandshake::DEFAULT_EXTENSIONS
                | checkhandshake::SRP_CLI_EXTENSION,
                "SRP extension test");
+
+#Test 21: EC handshake
+SKIP: {
+    skip "No EC support in this OpenSSL build", 1 if disabled("ec");
+    $proxy->clear();
+    $proxy->clientflags("-no_tls1_3");
```

Comment:
Is that flag strictly necessary?  I would prefer if it's avoided as much as possible.  My experience with sslvertol has shown that it's not as necessary as it was once...